moped-editor/package-lock.json

```diff
@@ -115,25 +115,27 @@
       }
     },
     "node_modules/@apollo/client": {
-      "version": "3.8.9",
-      "resolved": "https://registry.npmjs.org/@apollo/client/-/client-3.8.9.tgz",
-      "integrity": "sha512-IcQDFEEPc9+PEQsxhxQvsoQ04BRarOzi/Ila5PcniRSDeKJWgY22dnp6+V1i1fWXRDVd1ybdvze4sFESDVQUCQ==",
+      "version": "3.10.4",
```

Comment:
I read the changelog up to 3.10.4 which the update in the package.json resolved to here. Nothing stuck out to me! 🚀